### PR TITLE
hide admin list from lower level admins

### PR DESCRIPTION
**Pull request by Peter-Springer**

### CODE REVIEW

**Comment by Peter-Springer** [comment]
As the list of "power" admins grows we can refactor this into an array but for now this update hides the admin list button from users who do not have access to create a new admin

**Comment by Peter-Springer** [comment]
Only certain users being able to create admins is a result of updating our firebase rules.

File: lib/components/Admin.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ export default class Admin extends Component {
       filter: 'all',
       dateFilter: 'all',
       adminList: false,
+      adminCheck: []
```
Comment (kerrd89) [inline lib/components/Admin.js]: are we using this anywhere?

**Comment by Peter-Springer** [comment]
Requested changes have been made. We only hide the input field and submit button now not the entire list.

**Comment by kerrd89** [comment]
I can confirm I don't see the input or the button, but I can still see the buttons to delete admins.

**Comment by Peter-Springer** [comment]
Good call, Forgot about those try now.

**Pull request merged**